quickwit-actors/src/envelope.rs

```diff
@@ -44,6 +44,14 @@ impl<A: Actor> Envelope<A> {
         self.0.message()
     }
 
+    pub fn message_typed<M: 'static>(&mut self) -> Option<M> {
```

Comment:
Sorry for the noise, I started writing a test that needed `message_typed`, and ended up removing it, but I left this method.